editor/src/components/canvas/canvas-strategies/interaction-state.ts

```diff
@@ -26,7 +26,7 @@ export interface DragInteractionData {
 }
 
 interface KeyboardInteractionData {
-  type: 'KEYBOARD'
+  type: 'KEYBOARD_ARROW'
```

Comment:
Why's this becoming `KEYBOARD_ARROW`?

Reply:
The idea was that because we only create this interaction for arrow keys, this is not a general keyboard interaction data, but only for moving with the arrows.
This is more relevant in my other PR, where we store the necessary state for moving elements there.
https://github.com/concrete-utopia/utopia/pull/2151